docs/running-on-kubernetes.md

```diff
@@ -507,6 +507,13 @@ See the [configuration page](configuration.html) for information on Spark config
     configuration properties e.g. <code>spark.kubernetes.namespace</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.internal.master</code></td>
```

Comment:
I suggest changing the name of this to `spark.kubernetes.driver.master`, which makes it clear that this is the one used by the driver.

Reply:
done